installer/install.iss

```diff
@@ -3332,6 +3367,25 @@ begin
         InstallWindowsTerminalFragment();
     end;
 
+    {
+        Optionally "skip" installing bundled SSH binaries conflicting with external OpenSSH:
+    }
+
+    if RdbSSH[GS_ExternalOpenSSH].Checked then begin
+        WizardForm.StatusLabel.Caption:='Removing bundled Git OpenSSH binaries';
+        Executables:=ListOpenSSHExecutables();
+        for I := 0 to GetArrayLength(Executables) - 1 do
+        begin
+            E := Executables[I];
+            // Only skip installation of binaries that are actually on the path already
+            if (FileSearch(E, GetEnv('PATH')) <> '') and
+               (not
```

Comment:
Please don't invent a new coding style here, but imitate how things are done elsewhere in `install.iss`. There is never a line break after a `not`.

Reply:
Sorry about that; will fix this.